src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid implements Figure {
```

Comment:
The class IsoscelesTrapezoid should not start with an empty line. Please remove the empty line before the class declaration.